insonmnia/hub/acl.go

```diff
@@ -32,6 +35,34 @@ func NewACLStorage() ACLStorage {
 	}
 }
 
+func (s *workerACLStorage) MarshalJSON() ([]byte, error) {
+	if s == nil {
+		return json.Marshal(nil)
+	}
+	s.mu.RLock()
```

Comment:
This will deadlock if is called under `Apply` (which is https://github.com/sonm-io/core/pull/182/files#diff-882aa38dff4c0888c4d85060cd652ac7L190).